frontend/views/utils/webShare.js

```diff
@@ -0,0 +1,3 @@
+export function activateWebShare (data: { title?: string, url?: string, text?: string }) {
+  navigator.share(data)
+}
```

Comment:
Because this file doesn't have a newline at the end the build fails. I'm also not thrilled with the idea of a file for a single line of code. I think this should be removed

Reply:
fixed!